app/views/govuk_publishing_components/components/_checkbox.html.erb

```diff
@@ -1,22 +1,26 @@
 <%
+  index ||= false
   id ||= "checkbox-#{SecureRandom.hex(4)}"
-  index ||= rand(1..100)
+  id = "#{id}-#{index}" if index
```

Comment:
I'm not sure where the checkbox component is currently used - is this change (passed id's don't have a random number assigned to them any more) likely to break anything anywhere?

Reply:
Might be worth moving this id change out of the commit which adds the data attributes in case, for any reason, we need to revert one but not both of these changes in the future.

Reply:
I checked with @DilwoarH and this should be okay. The index is added for when the component is used in the checkboxes component, which passes both id and index to the checkbox component.